src/actions/merged.ts

```diff
@@ -19,7 +19,7 @@ export async function merged(
 
     const balanceSheet = await points.readBalanceFile(basePath)
     const oldBalance =
-        (balanceSheet && points.readBalances(balanceSheet)[user.id]) || 0
+        (balanceSheet && points.readBalances(balanceSheet)[user.id]) || 100
```

Comment:
```suggestion
        (balanceSheet && points.readBalances(balanceSheet)[user.id]) ?? 100
```

Otherwise 0 GBP will turn into 100 because 0 is falsy.